solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -113,6 +113,15 @@ contract RandomBeacon is Ownable {
     ///         operator affected.
     uint256 public relayEntryTimeoutNotificationRewardMultiplier;
 
+    /// @notice Percentage of the staking contract malicious behavior
+    ///         notification reward which will be transferred to the notifier
+    ///         reporting about unauthorized signing. Notifiers are rewarded
+    ///         from a notifiers treasury pool. For example, if a
```

Comment:
> Notifiers are rewarded from a notifiers treasury pool. 

Can we update docs for `relayEntryTimeoutNotificationRewardMultiplier` and `dkgMaliciousResultNotificationRewardMultiplier` to tell the same? Both versions are correct but I find the one in this PR better and having one version makes sense given that it's the same pool.

Reply:
sure 108abbf